src/Compiler/FSComp.txt

```diff
@@ -1536,6 +1536,7 @@ tcPassingWithoutNullToValueOptionOfObj,"You can create 'ValueSome value' directl
 tcPassingWithoutNullToNonNullAP,"You can remove this |Null|NonNull| pattern usage."
 tcPassingWithoutNullToNonNullQuickAP,"You can remove this |NonNullQuick| pattern usage."
 tcPassingWithoutNullTononNullFunction,"You can remove this `nonNull` assertion."
+3263,tcNullableToStringOverride,"With nullness checking enabled, overrides of .ToString() method must return a non-nullable string. You can handle potential nulls via the built-in string function."
```

Comment:
For clarity, would prefer
```suggestion
3263,tcNullableToStringOverride,"With nullness checking enabled, overrides of .ToString() method must return a non-nullable string. You can handle potential nulls via the built-in `string` function."
```

Reply:
This does not look good without markdown, I have not seen it used elsewhere apart from the `nonNull` above.